project/Build.scala

```diff
@@ -507,4 +512,16 @@ object Build extends sbt.Build {
         )
       )
   )
+
+  lazy val pipeline = Project(
+    id = "gearpump-experiments-pipeline",
+    base = file("experiments/pipeline"),
+    settings = commonSettings ++ myAssemblySettings ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hbase" % "hbase-client" % clouderaHBaseVersion,
+          "org.apache.hbase" % "hbase-common" % clouderaHBaseVersion
```

Comment:
Hi Kam,

I think it is good for example/streaming/kafka_hbase_pipeline project to depends on experiment module experiment/hbasesink.

How do you think?